### ISSUE TITLE
Bug: Tool not loading results

### ISSUE THREAD

**Issue by acohen84** [votinginfoproject/vip-embeddable-tool#175]
Can you confirm that I don't need to update the look-up tool code on the VIP website?  Because it's not loading results right now.

Also, if someone took the code off the website, will their tool work?  We have a high profile embed coming soon and I don't want this to fail.

Thanks!

**Comment by acohen84** [comment]
Address: 1500 Fairbanks Ave, Norfolk, VA

**Comment by acohen84** [comment]
+ @mariabianchi - this might be specific to Virginia.

**Comment by MariaBianchi** [comment]
I'm not seeing this. The tool loaded as expected for me with that address. We're talking about the one here: https://www.votinginfoproject.org/projects/vip-voting-information-tool/?polling-location?

**Comment by acohen84** [comment]
Here's a screenshot of what I see for 1500 Fairbanks Ave, Norfolk VA.  This is also what happens with another Virginia address.  Maria mentioned this was happening for her yesterday in staging data with Oklahoma addresses.  It seems to not be an urgent issue, but one to look into because I'd prefer the live tool not fail at random.

![blank screen](https://cloud.githubusercontent.com/assets/11788886/12833490/04e98ee8-cb6f-11e5-8e1a-79ce799d5b0a.jpg)

**Comment by MariaBianchi** [comment]
Like Amy said, I was seeing this same behavior with the copy of the tool embedded at hidden.votinginfoproject.org/new.html. Once I entered an Oklahoma address, the entire frame containing the embeddable tool went blank. Interestingly, this wasn't happening if I entered addresses in other states. I noticed the problem shortly after 11am yesterday, and by 11am today it seemed to be resolved without my having done anything.  I'm using Chrome on a Mac, and the behavior was happening in Safari as well.
